src/library/librarycontrol.cpp

```diff
@@ -398,7 +398,9 @@ void LibraryControl::emitKeyEvent(QKeyEvent&& event) {
         }
     }
     // Send a pointer to the event to focused widget
-    QApplication::sendEvent(focusWidget, &event);
+    for (auto i = 0; i < event.count(); ++i) {
```

Comment:
Hm, I guess count isn't used in QAbstractItemView which explains it:
https://code.qt.io/cgit/qt/qt.git/tree/src/gui/itemviews/qabstractitemview.cpp#n2246
I hope it doesn't start working in the future :).